l10n.yaml

```diff
@@ -3,5 +3,6 @@ template-arb-file: app_en.arb
 output-localization-file: breez_translations.dart
 output-class: BreezTranslations
 output-dir: lib/generated
+preferred-supported-locales: ["en"]
```

Comment:
What is the function of this change?

Reply:
Ok; I just realized that is the change you mentioned that is fixing the English as first in the languages order; that is amazing; thank you very much :rocket: